### PR TITLE
[DH-53] edx deployment Configuration

### PR DESCRIPTION
**Pull request by sean-morris**
- Configured edx hub
- Removed data8x from repo
- Removed data8xv2 from repo

### CODE REVIEW

File: .DS_Store
Comment (shaneknapp) [inline .DS_Store:1]: could you please `git rm` this file, and add the following entry to `datahub/.gitignore`:  `**/.DS_Store`.  thanks!

i can't believe we haven't done this yet...  :eyeroll:

File: deployments/edx/image/environment.yml
Comment (shaneknapp) [inline deployments/edx/image/environment.yml:1]: could you bump the versions of jupyterlab and notebook to latest/greatest?

File: deployments/edx/image/environment.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - ipywidgets==8.0.7
+  # disable until fixed (probably this:  https://github.com/jupyterlab/jupyter-collaboration/issues/162)
+  # - jupyter_collaboration==1.0.1
+  - jupyterhub==4.0.2
+  - jupyterlab-accessible-themes==0.2.0
+  - jupyter-archive==3.4.0
+  - nbconvert==7.6.0
+  - pytest-notebook==0.8.1
+  - gh-scoped-creds==4.1
+  - popularity-contest==0.4.1
```
Comment (shaneknapp) [inline deployments/edx/image/environment.yml]: change this to , and add `git+https://github.com/shaneknapp/python-popularity-contest.git@add-error-handling`

**Review by shaneknapp — CHANGES_REQUESTED** [review]
first pass, but looking really good!

**Review by shaneknapp — CHANGES_REQUESTED** [review]
could you also add in the circleci bits, and remove the vestiges of data8x from the [github labeler](https://github.com/berkeley-dsep-infra/datahub/blob/staging/.github/labeler.yml)?


i thought a separate PR would suffice, but it should probably live in this one.  you can copy what i did in https://github.com/berkeley-dsep-infra/datahub/pull/5241

File: deployments/edx/config/common.yaml [hunk 2]
```diff
@@ -1,9 +1,3 @@
-nfsPVC:
-  enabled: true
-  nfs:
-    serverIP: 10.128.0.62
-    shareName: export/data8xv2homes-2022-02-16
-
 jupyterhub:
```
Comment (shaneknapp) [inline deployments/edx/config/common.yaml:5]: you should add a common NFS block at the top of this file containing:

```
nfsPVC:
  enabled: true
  nfs:
    serverIP: 10.128.0.62
```

File: deployments/edx/config/prod.yaml [hunk 3]
```diff
@@ -1,15 +1,20 @@
+nfsPVC:
```
Comment (shaneknapp) [inline deployments/edx/config/prod.yaml:1]: this block (in both prod and staging) should look like this:

```
nfsPVC:
  nfs:
    export/edxhomes-2023-08-28/prod
```

s/prod/staging/ for `staging.yml`.

File: deployments/edx/config/staging.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+nfsPVC:
```
Comment (shaneknapp) [inline deployments/edx/config/staging.yaml:1]: see comment above in `prod.yaml`.

**Review by shaneknapp — CHANGES_REQUESTED** [review]

File: deployments/edx/config/common.yaml (hunk 2 above)
Comment (sean-morris) [reply deployments/edx/config/common.yaml:5]: done good deal

**Comment by sean-morris** [comment]
> could you also add in the circleci bits, and remove the vestiges of data8x from the [github labeler](https://github.com/berkeley-dsep-infra/datahub/blob/staging/.github/labeler.yml)?
> 
> i thought a separate PR would suffice, but it should probably live in this one. you can copy what i did in #5241

Ok, I removed the data8x pieces from the github labeler and added in the edx pieces to circle ci

**Review by shaneknapp — APPROVED** [review]

**Pull request merged**